internal/resource/config/configmap_builder.go

```diff
@@ -314,6 +315,25 @@ func (b *ConfigmapBuilder) Update(object client.Object) error {
 				Tags: map[string]string{"type": "{{ .Env.SERVICES }}"},
 			},
 		}
+
+		if b.instance.Spec.Metrics.PerUnitHistogramBoundaries != nil {
+			buckets := make(map[string][]float64)
+			p := b.instance.Spec.Metrics.PerUnitHistogramBoundaries
+			//convert map[string][]string to map[string][]float64
+			for key, value := range p {
+				var floatSlice []float64
+				for _, str := range value {
+					floatVal, err := strconv.ParseFloat(str, 64)
+					if err != nil {
+						return fmt.Errorf("can't build metrics config: Error converting %s to float: %v\n", str, err)
```

Comment:
```suggestion
						return fmt.Errorf("can't build metrics config: Error converting %s to float: %w\n", str, err)
```
The convention in this project seems to be `%w` for errors.